makefile

```diff
@@ -1,6 +1,10 @@
+ifeq (${BUILD},RELEASE)
+OPTIMIZATION=-O3
+endif
+
```

Comment:
I think Jenkins will end up using default so debug build so that would be not desirable.